warp10/src/main/java/io/warp10/continuum/Tokens.java

```diff
@@ -329,6 +332,10 @@ public static ReadToken extractReadToken(String token) throws WarpScriptExceptio
       }
     }
 
+    if (!labels.isEmpty()) {
+      labelSelectors.putAll(labels);
```

Comment:
Right, I've pushed a new version w/ labels being read first, and preventing owner/app/producer being forged into extra labels. Btw it breaks the ability to scope a wildcard read token through its extra labels. (not sure it has any sense anyway)